supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActorSpec.scala

```diff
@@ -423,4 +453,11 @@ class HtCondorJobExecutionActorSpec extends TestKitSuite("HtCondorJobExecutionAc
     override def storeExecutionResult(cachedExecutionResult: CachedExecutionResult): Unit = ()
   }
 
+  class KVServiceActor extends Actor {
```

Comment:
Not that it's much important in the scope of writing tests (at least for me personally), a TestProbe could have been a better candidate for a Test Actor this simple in functionality.